### PR TITLE
[FIX] Transaction amount in popup window incorrect

### LINKED ISSUE #768: Transaction amount in popup window incorrect
When substract fee from amount is checked ,the pop up window for tx confirmation does show substract fee from amount. The actual tx is correct but the display is wrong.

Steps to replicate:
Send a tx with substract fee from amount box checked

image

### PR DESCRIPTION
**Pull request by aguycalled**
Fixes #768.

### CODE REVIEW

File: src/qt/sendcoinsdialog.cpp [hunk 1]
```diff
@@ -321,7 +328,7 @@ void SendCoinsDialog::on_sendButton_clicked()
 
     // add total amount in all subdivision units
     questionString.append("<hr />");
-    CAmount totalAmount = nTotalAmount + txFee;
+    CAmount totalAmount = nTotalAmount + (rcp.fSubtractFeeFromAmount ? -txFee : txFee);
```
Comment (mxaddict) [inline src/qt/sendcoinsdialog.cpp:331]: should this be ?:
```cpp
CAmount totalAmount = nTotalAmount + (rcp.fSubtractFeeFromAmount ? 0 : txFee);
```

Comment (aguycalled) [reply src/qt/sendcoinsdialog.cpp:331]: without subtracting fee:

image

with subtracting fee:

image

Comment (mxaddict) [reply src/qt/sendcoinsdialog.cpp:331]: The total amount should be equal to 1 NAV right? Cause the fee is subtracted from the amount, but it's still needed.

Comment (aguycalled) [reply src/qt/sendcoinsdialog.cpp]: fixed in last commit:

without subtracting:

image

with subtracting:

image

Comment (mxaddict) [reply src/qt/sendcoinsdialog.cpp]: > fixed in last commit:
> 
> without subtracting:
> 
> image
> 
> with subtracting:
> 
> image

Nice, will test in a few minutes

**Comment by chasingkirkjufell** [comment]
away from my linux node atm, will test gitian

**Review by mxaddict — APPROVED** [review]
image


image


Looks good.

**Comment by navbuilder** [comment]
A new build of 65eb7b7856c292a2c962ce39f16b0246f588136b has completed succesfully!
Binaries available at https://build.nav.community/binaries/fix-subtract-fee

**Review by chasingkirkjufell — APPROVED** [review]
Confirmed fix with gitian build on windows 10

**Pull request merged**